src/emc/rs274ngc/interp_arc.cc

```diff
@@ -28,6 +28,10 @@
 #include "rs274ngc_interp.hh"
 #include "interp_internal.hh"
 
+#ifndef M_PI_2l
```

Comment:
Instead of adding this in many source files, I would prefer to see it added to src/rtapi/rtapi_math.h, and add inclusions of <rtapi_math.h> as necessary.